Dockerfile

```diff
@@ -1,14 +1,13 @@
 ### Build stage for the website frontend
-FROM --platform=$BUILDPLATFORM node:20-bullseye-slim as build
+FROM --platform=$BUILDPLATFORM node:20-bookworm-slim as build
 RUN apt-get update && \
-apt-get install -y python
+apt-get install -y python3
 WORKDIR /code
 COPY . ./
 RUN npm install
 RUN npm run build
 
-FROM nginx:1.25.3-alpine
+FROM nginx:1.25.4-alpine
 COPY --from=build /code/build/ /usr/share/nginx/html
-COPY --from=build /code/config.json /usr/share/nginx/html/config.json
-
-EXPOSE 80
\ No newline at end of file
+COPY --from=build /code/config*.json /usr/share/nginx/html/
```

Comment:
i think we should copy the [config.example.json](https://github.com/freifunkMUC/meshviewer/blob/main/config.example.json)?